helper.py

```diff
@@ -0,0 +1,30 @@
+# Copies files and folders to relevant containers
+import os
+
+
+def cp_helper(name, test_name, container):
+    if test_name == "storage":
+        dest = "scratch/" + name + "/"
```

Comment:
This is very prone to error, together with line #21, #27
Please, use os.path.join